CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.6.2.0.1
+- On initialization, always set ageRestrictedUser to true.
```

Comment:
Nit: Maybe also say that this is at HyprMX's request. Otherwise, this change does not make much sense by itself.

Reply:
Fair point. 👍🏼 Will adjust.